deploy/crds/Grafana.yaml

```diff
@@ -127,6 +127,9 @@ spec:
                 priorityClassName:
                   type: string
                   description: Pod priority class name
+            skipCreateServiceAccount:
```

Comment:
why not move it into the `serivceAccount` object and call it `skip`?

Reply:
Yup, on second thought that'll be the best place for it